src/AI/Muriel.java

```diff
@@ -457,15 +457,15 @@ private boolean queueUnitAction(GameMap gameMap, Unit unit, boolean allowDeferri
         Unit unitInTheWay = gameMap.getResident(action.getMoveLocation()); // Could be us, could be nobody.
 
         // Sift through all attack actions we can perform.
-        double damageValue = AICombatUtils.scoreAttackAction(unit, action, gameMap,
+        int damageValue = AICombatUtils.scoreAttackAction(unit, action, gameMap,
             (results) -> {
-              double healthDamage = Math.min(results.defender.getPreciseHealthDamage(), results.defender.unit.health);
+              int healthDamage = Math.min(results.defender.getPreciseHealthDamage(), results.defender.unit.health);
 
               if( shouldAttack(unit, results.defender.unit, gameMap) )
-                return (results.defender.unit.getCost() / 10) * healthDamage;
+                return (results.defender.unit.getCost() / UnitModel.MAXIMUM_HEALTH) * healthDamage;
```

Comment:
Shoulda known better than to use a magic number, even for something as "unchangeable" as unit max HP. Silly me.